mindjunkies/live_classes/tests/test_models.py

```
@@ -1,59 +1,61 @@
-def test_get_meeting_url_teacher(mocker):
-    """Test that `get_meeting_url_teacher()` returns correct JWT-secured URL."""
-    teacher = baker.make(User)
-    course = baker.make(Course)
-    live_class = baker.make(
-        LiveClass, teacher=teacher, course=course, scheduled_at=now()
+def test_get_meeting_url_student(settings):
+    settings.JITSI_APP_ID = "myapp"
+    live_class = baker.make(LiveClass, meeting_id="abc123")
+    assert (
+        live_class.get_meeting_url_student()
+        == f"https://8x8.vc/myapp/abc123"
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)